src/alitra/align_frames.py

```diff
@@ -57,18 +57,15 @@ def align_frames(
             from_=p_1.frame,
             to_=p_2.frame,
         )  # type: ignore
-
-        euler: Euler = Euler.from_array(
-            rotation_object.as_euler("zyx"),
+        transform = Transform(
             from_=p_1.frame,
             to_=p_2.frame,
+            translation=translations,
+            rotation_object=rotation_object,
         )
-        euler = AlignFrames._set_non_rotating_angles_to_zero(euler, rot_axes)
 
         try:
-            frame_transform = FrameTransform(
-                euler=euler, translation=translations, from_=p_1.frame, to_=p_2.frame
-            )
+            frame_transform = FrameTransform(transform)
```

Comment:
This should be type hinted. Especially important as this is the return variable of the function.